sdcm/tester.py

```diff
@@ -203,6 +203,11 @@ def __init__(self, methodName='test'):  # pylint: disable=too-many-statements
         time.sleep(0.5)
         InfoEvent('TEST_START test_id=%s' % cluster.Setup.test_id())
 
+    def run(self, result=None):
```

Comment:
Why did you add this?

Reply:
again, it was for having a way to add failures/errors into the tester object